app/database/server/schema/index.ts

```diff
@@ -0,0 +1,292 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {AppSchema, appSchema, tableSchema} from '@nozbe/watermelondb';
+import {MM_TABLES} from '@constants/database';
+
+export const serverSchema: AppSchema = appSchema({
+    version: 1,
+    tables: [
+        tableSchema({
+            name: MM_TABLES.SERVER.CHANNEL,
```

Comment:
this file can easily become super large and get out of hand... is there a way that you could just import the table schema for each table?

Reply:
Absolutely.  I think, we can have a 'table_schemas' folder which houses several files.  Each one containing one default export for a table schema.   This approach is better than having multiple exports in a single file.  Let me know if you approve this approach; I will tackle it in a few minutes.

Reply:
Go for it

Reply:
Thank you 👍

Reply:
Done.